platform/logging/layouts/JsonLayout.ts

```diff
@@ -0,0 +1,30 @@
+import { Schema, typeOfSchema } from '../../types';
+import { LogRecord } from '../LogRecord';
+import { Layout } from './Layouts';
+
+const createSchema = ({ literal, object }: Schema) => {
+  return object({
+    kind: literal('json')
+  });
+};
+
+const schemaType = typeOfSchema(createSchema);
+/** @internal */
+export type JsonLayoutConfigType = typeof schemaType;
+
+/**
+ * Layout that just converts `LogRecord` into JSON string.
+ * @internal
+ */
+export class JsonLayout implements Layout {
+  static createConfigSchema = createSchema;
+
+  format(record: LogRecord): string {
+    // Per spec `timestamp.toJSON()` uses `timestamp.toISOString()` under the hood.
+    // See http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.5.44
+    return JSON.stringify({
```

Comment:
++ I prefer the "more specific" format. We should also take a look at current Kibana to make sure we're "fairly consistent", but I'm good with doing that in a follow-up if you want to